packages/apps/helpdesk/src/screens/TicketDetailsScreen.js

```diff
@@ -0,0 +1,169 @@
+  Button,
+  useThemeColor,
+} from '@axelor/aos-mobile-ui';
+import {
+  Stopwatch,
+  useDispatch,
+  useSelector,
+  useTranslator,
+  getNowDateZonesISOString,
+} from '@axelor/aos-mobile-core';
+import {fetchTicketById, updateTicketStatus} from '../features/ticketSlice';
+import {TicketHeader, TicketDropdownCards, TicketBottom} from '../components';
+import {fetchTimerById, searchTimerHistoryById} from '../features/timerSlice';
+import {Ticket} from '../types';
+import {fetchHelpdeskConfigApi} from '../features/helpdeskConfigSlice';
+
+const TicketDetailsScreen = ({navigation, route}) => {
+  const {idTicket, colorIndex} = route.params;
+  const I18n = useTranslator();
+  const Colors = useThemeColor();
+
+  const dispatch = useDispatch();
+  const {ticket} = useSelector(state => state.ticket);
+  const {timer, timerHistory} = useSelector(state => state.timer);
+  const {helpdeskConfig} = useSelector(state => state.helpdeskConfig);
+
+  useEffect(() => {
+    dispatch(fetchHelpdeskConfigApi());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(fetchTicketById({ticketId: idTicket}));
+  }, [dispatch, idTicket]);
+
+  useEffect(() => {
+    if (ticket?.timerList?.length > 0) {
+      dispatch(fetchTimerById({timerId: ticket?.timerList[0]?.id}));
+    }
+  }, [dispatch, ticket?.timerList]);
+
+  useEffect(() => {
+    if (timer?.timerHistoryList?.length > 0) {
+      dispatch(searchTimerHistoryById({idTimer: timer?.id}));
+    }
+  }, [dispatch, timer]);
+
+  const timerInProgress = useMemo(() => {
+    if (
+      Object.keys(timer).length !== 0 &&
+      timer != null &&
+      timer?.statusSelect !== 0
+    ) {
+      return true;
+    }
+    return false;
+  }, [timer]);
+
+  const disbaled = useMemo(() => {
+    if (
+      ticket?.statusSelect === Ticket.status.Closed ||
+      ticket?.statusSelect === Ticket.status.Resolved
+    ) {
+      return true;
+    }
+    return false;
+  }, [ticket]);
+
+  const timerDuration = useMemo(() => {
+    return Ticket.getTotalDuration(timerHistory);
+  }, [timerHistory]);
+
+  const updateStatus = useCallback(
+    status => {
+      dispatch(
+        updateTicketStatus({
+          version: ticket?.version,
+          dateTime: getNowDateZonesISOString(),
+          targetStatus: status,
+          ticketId: ticket?.id,
+        }),
+      );
+    },
+    [dispatch, ticket],
+  );
+
+  if (ticket?.id !== idTicket) {
+    return null;
+  }
+
+  return (
+    <Screen removeSpaceOnTop={true}>
+      <HeaderContainer
+        expandableFilter={false}
+        fixedItems={<TicketHeader colorIndex={colorIndex} />}
+      />
+      <ScrollView>
+        <NotesCard
+          title={I18n.t('Base_Description')}
+          data={ticket.description}
+        />
+        <TicketDropdownCards />
+        {helpdeskConfig?.manageTimer ? (
+          <Stopwatch
+            startTime={timerDuration}
+            timerFormat={I18n.t('Stopwatch_TimerFormat')}
+            inProgressAtStart={timerInProgress}
```

Comment:
i thought we said we were removing this ugly props 🥺 Develop matcher in Ticket type file to get StopWatch status depending on Ticket state and use standard `status` props (inspire yourself from `OperationOrderStopwatch`, that's the same principle)